test/e2e/pivoting.go

```diff
@@ -86,23 +87,24 @@ func pivoting(ctx context.Context, inputGetter func() PivotingInput) {
 	if ephemeralCluster == Kind {
 		isIronicDeployment = false
 	}
-	removeIronic(ctx, func() RemoveIronicInput {
-		return RemoveIronicInput{
+	removeIronicBMO(ctx, func() RemoveIronicBMOInput {
+		return RemoveIronicBMOInput{
 			ManagementCluster: input.BootstrapClusterProxy,
 			IsDeployment:      isIronicDeployment,
 			Namespace:         input.E2EConfig.GetVariable(ironicNamespace),
 			NamePrefix:        input.E2EConfig.GetVariable(NamePrefix),
+			NameSuffix:        "-ironic",
 		}
 	})
 
 	By("Create Ironic namespace")
 	targetClusterClientSet := input.TargetCluster.GetClientSet()
-	ironicNamespace := &corev1.Namespace{
+	iNamespace := &corev1.Namespace{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: input.E2EConfig.GetVariable(ironicNamespace),
 		},
 	}
-	_, err := targetClusterClientSet.CoreV1().Namespaces().Create(ctx, ironicNamespace, metav1.CreateOptions{})
+	_, err := targetClusterClientSet.CoreV1().Namespaces().Create(ctx, iNamespace, metav1.CreateOptions{})
```

Comment:
Oh I see! What if we rename `ironicNamespace` -> `ironicNamespaceName` or `ironicNamespaceVar` (since it is the variable where we get the name from) and then this can be `ironicNamespace`?